docs/topics_5/rn-known-issues.adoc

```diff
@@ -12,6 +12,14 @@ At the time of release, the following known issues are identified as major issue
 |Component
 |Summary
 
+|link:https://issues.redhat.com/browse/WINDUP-2704[WINDUP-2704]
+|Web UI & Windup-as-a-Service
+|The `run_mta.sh` script tries to set the soft `ulimit` to 100000 which causes an issue on macOS.
```

Comment:
rewrite: 

The value of the `maxfiles` parameter on macOS must be `10000` because the `run_mta.sh` script sets the soft limit to `10000`.

I would do the same for the mta-cli known issue